lib/src/Pos/Util/Swagger.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Pos.Util.Swagger where
+
+import           Universum
+
+import           Data.Swagger
+import           NeatInterpolation (text)
+import           Servant.Server (Handler, Server)
+import           Servant.Swagger.UI.Core (SwaggerSchemaUI',
+                     swaggerSchemaUIServerImpl)
+import           Servant.Swagger.UI.ReDoc (redocFiles)
+
+-- | Provide an alternative UI (ReDoc) for rendering Swagger documentation.
+swaggerSchemaUIServer
+    :: (Server api ~ Handler Swagger)
+    => Swagger -> Server (SwaggerSchemaUI' dir api)
+swaggerSchemaUIServer =
+    swaggerSchemaUIServerImpl redocIndexTemplate redocFiles
+  where
+    redocIndexTemplate :: Text
+    redocIndexTemplate = [text|
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>ReDoc</title>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+      body { margin: 0; padding: 0; }
+    </style>
+    <script>
+        // Force Strict-URL Routing for assets relative paths
+        (function onload() {
+            if (!window.location.href.endsWith("/")) {
+                window.location.href += "/";
+            }
+        }());
+    </script>
+  </head>
+  <body>
+    <redoc spec-url="../SERVANT_SWAGGER_UI_SCHEMA"></redoc>
+    <script src="redoc.min.js"> </script>
+  </body>
+</html>|]
```

Comment:
NOTE: upstream PR has been accepted some weeks ago and the default template has been fixed. So, using
`servant-swagger-ui-redoc-0.3.2.1.22.2` should make this obsolete; using `redocSchemaUIServer` would be enough :smile: